app.vue

```diff
@@ -28,7 +17,6 @@ function useAppSeo() {
   useHead({
     titleTemplate: (c) => c ? `${c} - Alexander Lichter` : 'Alexander Lichter - Web Engineering Consultant',
     link: [
-      { rel: 'canonical', href: canonical },
```

Comment:
When you use `@nuxtjs/seo`, it will apply these defaults for you https://nuxtseo.com/nuxt-seo/guides/default-meta